### PR TITLE
update readme copy

### PR DESCRIPTION
**Pull request by tuves**

### CODE REVIEW

**Review by jenna-wolf — CHANGES_REQUESTED** [review]
Referencing [my feedback](https://github.com/github-community/community/pull/20600/commits/197c30c3d614435d0c7014668e4a23c6f3cceb3d#r922506657) to a URL here, I think this applies to all URLs in this PR referencing the new org name. Should we just have at top level org discussions, or directly in the repo Discussions? The org level discussions will look nicer. 

@ettaboyle @tuves @michellemerrill @natalierjackson

File: README.md [hunk 1]
```diff
@@ -1,36 +1,52 @@
...
-| 🚢  [Actions and Packages](https://github.com/github-community/community/discussions/categories/actions-and-packages) 	| [GitHub Actions](https://github.com/features/actions) and [GitHub Packages](https://github.com/features/packages) |
-| 🔎  [Code Search & Navigation](https://github.com/github-community/community/discussions/categories/code-search-and-navigation) 	| [Code Search & Navigation](https://cs.github.com/about) 	|
-| 💻  [Codespaces](https://github.com/github-community/community/discussions/categories/codespaces) 	| [GitHub Codespaces](https://github.com/features/codespaces) 	|
+| 👍  [Accessibility](https://github.com/community/discussions/categories/accessibility) 	| [About Accessibility](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-accessibility-settings#about-accessibility-settings) |
```
[the remark this answers is not in the document]
Comment (Aa0541599451) [reply README.md:9]: [](https://github.com/Aa0541599451/Aa0541599451-.git)

**Comment by Aa0541599451** [comment]
انا

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aa0541599451) [reply README.md:9]: - [ ] [](https://github.com/github-community/community/pull/20600#pullrequestreview-1040791342)
- - 
@Aa0541599451 @https://github.com/github-community/community/pull/20600#pullrequestreview-1040791342

**Review by MohamudMmc — COMMENTED** [review]
👍

**Review by zayidmohamedy — COMMENTED** [review]
👍

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ettaboyle) [reply README.md:9]: https://github.com/orgs/community/discussions/categories/announcements is an example of the actual URL structure post org rename for category links

File: README.md [hunk 2]
```diff
@@ -1,36 +1,52 @@
...
 
 ## Making suggestions
 
-We encourage you to [open a discussion](https://github.com/github-community/community/) if you have suggestions for how we can improve our products. You don't need to have a solution to the problem you are facing to kick off a discussion. We are hoping to foster productive and collaborative conversations, so please check out [how to give good feedback](https://github.com/github-community/community/discussions/1) if you want some guidance on how to kick off a successful discussion.
+We encourage you to [open a discussion](https://github.com/community) if you have suggestions for how we can improve our products. You don't need to have a solution to the problem you are facing to kick off a discussion. We hope to foster productive and collaborative conversations, so please check out [how to give good feedback](https://github.com/community/discussions/1) for guidance on how to kick off a successful discussion.
```
Comment (ettaboyle) [inline README.md:47]:
```suggestion
We encourage you to [open a discussion](https://github.com/orgs/community/discussions) if you have suggestions for how we can improve our products. You don't need to have a solution to the problem you are facing to kick off a discussion. We hope to foster productive and collaborative conversations, so please check out [how to give good feedback](https://github.com/community/discussions/1) for guidance on how to kick off a successful discussion.
```

**Review by ettaboyle — APPROVED** [review]
Approved

**Comment by ettaboyle** [comment]
[HOLD] Do not merge until we update org url

**Comment by Mystic421980** [comment]
> [HOLD] Do not merge until we update org url

SigTerm

**Comment by Mystic421980** [comment]
> > [HOLD] Do not merge until we update org url
> 
> SigTerm

Run

**Review by ettaboyle — APPROVED** [review]

**Pull request merged**